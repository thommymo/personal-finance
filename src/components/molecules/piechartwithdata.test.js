import React from 'react';
import ReactDOM from 'react-dom';
import PieChartWithData from './piechartwithdata';

it('renders without crashing', () => {
  const div = document.createElement('div');
  ReactDOM.render(<PieChartWithData />, div);
});
