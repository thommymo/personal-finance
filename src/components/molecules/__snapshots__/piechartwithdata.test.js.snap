// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Shows portfolio as pie chart 1`] = `
<div>
  <div
    className="sc-bdVaJa eAwGjB"
  >
    <h1
      className="sc-htpNat cfuYBV"
    >
      Your Portfolio
    </h1>
    <h5
      className="sc-bxivhb fuZUuW"
    >
      CHF302,462.82
    </h5>
  </div>
  <div
    className="chart "
  />
</div>
`;
